fastddsgen -python Build Error [17648]

### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

```
struct Test{
    sequence<sequence<double> >  data;
};
```
use command
```
fastddsgen -pyhton xx.idl  
cmake . 
make
```
it‘s should normal build,and Cpp is built successfully, but python is wrong

### Current behavior

```
root@8ae453f9e855:~/Fast-DDS/test/2/py# cmake .
-- Configuring python wrapper for types in test...
-- The C compiler identification is GNU 7.5.0
-- The CXX compiler identification is GNU 7.5.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libcrypto.so (found version "1.1.1f")  
-- Found SWIG: /usr/local/bin/swig (found version "4.1.1") 
-- Found Python3: /usr/bin/python3.8 (found version "3.8.10") found components: Interpreter Development 
-- Configuring done
-- Generating done
-- Build files have been written to: /root/Fast-DDS/test/2/py
root@8ae453f9e855:~/Fast-DDS/test/2/py# make
Scanning dependencies of target testWrapper_swig_compilation
[ 16%] Swig compile test.i for python
/usr/local/include/fastdds/dds/core/LoanableTypedCollection.hpp:68: Warning 389: operator[] ignored (consider using %extend)
/usr/local/include/fastdds/dds/core/LoanableTypedCollection.hpp:93: Warning 389: operator[] ignored (consider using %extend)
/root/Fast-DDS/test/2/py/test.i:68: Error: Syntax error in input(1).
make[2]: *** [CMakeFiles/testWrapper_swig_compilation.dir/build.make:64: CMakeFiles/testWrapper.dir/testPYTHON.stamp] Error 1
make[2]: *** Deleting file 'CMakeFiles/testWrapper.dir/testPYTHON.stamp'
make[1]: *** [CMakeFiles/Makefile2:79: CMakeFiles/testWrapper_swig_compilation.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

### Steps to reproduce

```
struct Test{
    sequence<sequence<double> >  data;
};
```
use command
```
fastddsgen -pyhton xx.idl
cmake .
make
```

### Fast DDS version/commit

v2.8.0

### Platform/Architecture

Ubuntu Focal 20.04 amd64

### Transport layer

Default configuration, UDPv4 & SHM

### Additional context

_No response_

### XML configuration file

_No response_

### Relevant log output

_No response_

### Network traffic capture

_No response_

--- a/src/main/java/com/eprosima/fastcdr/idl/templates/TypesSwigInterface.stg
+++ b/src/main/java/com/eprosima/fastcdr/idl/templates/TypesSwigInterface.stg
@@ -92,6 +92,22 @@ $struct.members : {$member_getters(struct_name=struct.name, member=it)$}; separa
 
 >>
 
+template_sequence_name(typecode) ::= <<
+$if(typecode.isSequenceType)$
+$template_sequence_name(typecode.contentTypeCode)$_vector
+$else$
+$typecode.cppTypename$_vector
+$endif$
+>>
+
+template_sequence(typecode) ::= <<
+$if(typecode.contentTypeCode.isSequenceType)$
+$template_sequence(typecode.contentTypeCode)$
+$endif$
+
+%template($template_sequence_name(typecode.contentTypeCode)$) std::vector<$typecode.contentTypeCode.cppTypename$>;
+>>
+
 member_getters(struct_name, member) ::= <<
 %ignore $struct_name$::$member.name$($member.typecode.cppTypename$&&);
 
@@ -100,7 +116,7 @@ member_getters(struct_name, member) ::= <<
 // We ignore them to prevent this
 $if(member.typecode.isSequenceType)$
 %ignore $struct_name$::$member.name$() const;
-%template($member.typecode.contentTypeCode.cppTypename$_vector) std::vector<$member.typecode.contentTypeCode.cppTypename$>;
+$template_sequence(member.typecode)$
 $elseif(member.typecode.isMapType)$
 %ignore $struct_name$::$member.name$() const;
 %template($member.typecode.keyTypeCode.cppTypename$_$member.typecode.valueTypeCode.cppTypename$_map) std::map<$member.typecode.keyTypeCode.cppTypename$,$member.typecode.valueTypeCode.cppTypename$>;

